app/views/content/blog/nqt-to-head-of-biology.md

```diff
@@ -0,0 +1,45 @@
+---
+title: From NQT to head of biology in 2 years
+date: "2019-11-18"
+author: Sarah Fisher
+images:
+  q-sarah:
+    path: "media/images/content/blog/stories-sarah-f.jpg"
+    thumbnail_path: "media/images/content/blog/stories-sarah-f.jpg"
+description: |-
+  Sarah Fisher shares her story of how she became a department head in a subject she loves.
+keywords:
+  - life as a teacher
+  - day in the life
+  - classroom
+tags:
+  - career progression
+---
+
+Since this blog post was first published newly qualified teacher (NQT) status has been replaced with early career teacher (ECT).
```

Comment:
```suggestion
Since this blog post was first published, the term newly qualified teacher (NQT) has been replaced with early career teacher (ECT). This describes a teacher in their first two years of teaching.
```
Not sure this phrasing is quite right? And could we make this bit more prominent, so it's not as integrated into the blog post? Maybe a smaller warning banner like what we're adding to the internships page?